runtime/gc_vlhgc/GlobalMarkCardScrubber.cpp

```diff
@@ -171,6 +177,37 @@ MM_GlobalMarkCardScrubber::scrubMixedObject(MM_EnvironmentVLHGC *env, J9Object *
 	return doScrub;
 }
 
+
+void
+stackSlotIteratorForGlobalMarkCardScrubber(J9JavaVM *javaVM, J9Object **slotPtr, void *localData, J9StackWalkState *walkState, const void *stackLocation)
+{
+	StackIteratorData4GlobalMarkCardScrubber *data = (StackIteratorData4GlobalMarkCardScrubber *)localData;
+	if (*data->doScrub) {
+		*data->doScrub = data->globalMarkCardScrubber->mayScrubReference(data->env, data->fromObject, *slotPtr);
+	}
+}
```

Comment:
It's unfortunate, but we probably cannot terminate iteration of slots once we do see for one slot that we cannot scurb (like we can terminate iteration of object slots). Leave a comment stating this.

I'd even consider forgoing scrubbing of cards having continuation objects, altogether. I believe it's not necessary for correctness, it's just an optimization that removes stale cards, in case all references point to the same region as the objects (within the card) itself.

In that case, scrubContinuationObject should just return false all the time. 

It's really hard to say how much it would be beneficial to scrub it, vs how much it would be waste of time iterating through all slots (and potentially finding out, it cannot be scrubbed).

I'll flip a coin and say let's leave the scrubbing in.